gcampus/core/models/measurement.py

```diff
@@ -254,3 +253,28 @@ def __str__(self):
             "pk": self.pk,
             "name": self.parameter_type.name,
         }
+
+
+class Calibration(models.Model):
+    class Meta:
+        verbose_name = _("Calibration")
+        verbose_name_plural = _("Calibrations")
+
+    name = models.CharField(blank=True, max_length=280, verbose_name=_("Name"))
+
+    calibration_parameter_type = models.ForeignKey(
+        ParameterType,
+        on_delete=models.PROTECT,
+        # We opted to use 'Parameter' in the front-end. See issue
+        # https://github.com/desklab/gcampus/issues/46 for more.
+        verbose_name=_("Parameter"),
+        related_name="calibration_parameter",
```

Comment:
I think the field name is a bit confusing. something like `parameter_type` should be sufficient. Same goes for the `related_name`: Can you use `calibrations` for that?

In the right context, the naming becomes redundant:

`calibration.parameter_type` is more descriptive then `calibration.calibration_parameter_type`

`parameter_type.calibrations` is more descriptive then `parameter_type.calibration_parameter`

Reply:
fixed in 067baa0b47adfd6587eff8d377fe351906522832